modules/virt-schedule-supported-cpu-model-vms.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-schedule-vms.adoc
+
+[id="virt-schedule-supported-cpu-model-vms_{context}"]
+= Scheduling virtual machines with the supported CPU model
+
+You can configure a CPU model for a virtual machine (VM) or a virtual machine instance (VMI) to schedule it on a node where its CPU model is supported.
+
+.Procedure
+
+. Edit the `domain` spec of your virtual machine configuration file. The following example shows a specific CPU model defined for a (VMI):
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+   name: myvmi
```

Comment:
Indentation is wrong here as well, there's an extra space for all `metadata` fields, and `cpu` should be within `domain`

Reply:
Indentation was adjusted. Yaml was updated.